fdbclient/FileBackupAgent.actor.cpp

```diff
@@ -1798,6 +1798,10 @@ struct BackupRangeTaskFunc : BackupTaskFuncBase {
 				// Initialize range file writer and write begin key
 				if (encryptionEnabled) {
 					CODE_PROBE(true, "using encrypted snapshot file writer");
+					if (!tenantCache.isValid()) {
```

Comment:
should this be an assert?

Reply:
No i changed it so we only initialize the cache when encryption is enabled